packages/rnv/src/systemTools/prompt.js

```diff
@@ -9,7 +9,10 @@ export const inquirerPrompt = async (params) => {
     const c = Config.getConfig();
     const msg = params.logMessage || params.warningMessage || params.message;
     if (c.program.ci) {
-        throw new Error(`--ci option does not allow prompts: ${msg}`);
+        if (Array.isArray(params.choices) && typeof params.default !== 'undefined' && params.choices.includes(params.default)) {
+            return Promise.resolve({ [params.name]: params.default });
+        }
+        throw new Error(`--ci option does not allow prompts. Consider adding a default to the ${params.name} question: ${msg}.`);
```

Comment:
misleading suggestion.  "Consider adding a default to the ${params.name} question"
users of RNV cannot change internals of it